EIPS/eip-3978.md

```diff
@@ -0,0 +1,52 @@
+---
+eip: 3978
+title: Gas refunds on reverts
+author: Anton Bukov (@k06a), Mikhail Melnik (@ZumZoom)
+discussions-to: <to be assigned>
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-09-16
+---
+
+## Simple Summary
```

Comment:
This section has been replaced with the `description` field in the preamble. Please see the `eip-template.md` for more info.